Automattic/wp-calypso
# Replace some of the support links for custom DNS records

Replace the support links for custom DNS records with the relevant sub-pages in the support docs. Also get rid of hardcoded URLs and replace them with constants.

## Proposed Changes

* Replace links in multiple places to point to the more specific support page

## Why are these changes being made?

* The custom DNS records support page was getting huge so we've broken it into smaller sub-pages and we can point customers to those pages instead of the huge support doc

## Testing Instructions

* In Calypso verify that the links for default records + the links in the add/edit pages are pointed to the correct support page

## Pre-merge Checklist
